From 83a2ca9fa67968905d01bcd3a232eaa2f6ed6d19 Mon Sep 17 00:00:00 2001
From: "Robert S. Edmonds" <edmonds@debian.org>
Date: Tue, 12 Aug 2014 15:38:40 -0400
Subject: [PATCH] control: Update protobuf-c build-dep.

collectd-dev ships .pb-c.h files generated by the protobuf-c compiler.
These need to be refreshed due to the protobuf-c transition.

Closes: #756422
---
 debian/changelog | 9 +++++++++
 debian/control   | 2 +-
 2 files changed, 10 insertions(+), 1 deletion(-)

diff --git a/debian/changelog b/debian/changelog
index fb49335..248fbb1 100644
--- a/debian/changelog
+++ b/debian/changelog
@@ -1,3 +1,12 @@
+collectd (5.4.1-3.1) unstable; urgency=medium
+
+  * Non-maintainer upload.
+  * debian/control:
+    - Build-Depend on libprotobuf-c-dev instead of libprotobuf-c0-dev due to
+      the protobuf-c transition (Closes: #756422).
+
+ -- Robert S. Edmonds <edmonds@debian.org>  Tue, 12 Aug 2014 15:36:27 -0400
+
 collectd (5.4.1-3) unstable; urgency=medium
 
   * debian/control:
diff --git a/debian/control b/debian/control
index e155c1c..00dba92 100644
--- a/debian/control
+++ b/debian/control
@@ -26,7 +26,7 @@ Build-Depends: debhelper (>= 5), dpkg-dev (>= 1.14.10), po-debconf, dpatch,
  libpcap0.8-dev | libpcap-dev,
  libperl-dev,
  libpq-dev,
- libprotobuf-c0-dev,
+ libprotobuf-c-dev,
  librabbitmq-dev,
  librrd-dev (>= 1.4~),
  libsensors4-dev [linux-any],
-- 
2.0.0

